hail/src/main/scala/is/hail/expr/ir/Pretty.scala

```diff
@@ -252,7 +252,7 @@ object Pretty {
             case AggLet(name, _, _, isScan) => prettyIdentifier(name) + " " + prettyBooleanLiteral(isScan)
             case TailLoop(name, args, _) => prettyIdentifier(name) + " " + prettyIdentifiers(args.map(_._1).toFastIndexedSeq)
             case Recur(name, _, t) => prettyIdentifier(name) + " " + t.parsableString()
-            case Ref(name, _) => prettyIdentifier(name)
+            case Ref(name, _) => prettyIdentifier(name) + " " + ir.typ.toString
```

Comment:
Let's back this out for legibility in the logs?

Reply:
oh, this is invalid anyway